src/citrine/resources/ara_definition.py

```diff
@@ -51,15 +52,17 @@ def _get_dups(lst: List) -> List:
 
     def __init__(self, *, name: str, description: str, datasets: List[UUID],
                  variables: List[Variable], rows: List[Row], columns: List[Column],
-                 uid: Optional[UUID] = None, version: Optional[int] = None):
+                 version_uid: Optional[UUID] = None, version_number: Optional[int] = None,
```

Comment:
@alexandraj777 here is were we define the identifier/version system for versioned assets (like modules and workflows, in the future)